litmus/compile.ml

```diff
@@ -104,6 +105,7 @@ module Generic (A : Arch_litmus.Base)
                     | MiscParser.TyDefPointer -> Some pointer
                     | MiscParser.Ty s -> Some (Base s)
                     | MiscParser.Atomic s -> Some (Atomic (Base s))
+                    | MiscParser.Address -> None
```

Comment:
May be `assert false` or `Warn.fatal ...`. Not important. This "typing" code is not as armored as it should anyway.

Reply:
ok added